ui/src/pages/Upload.tsx

```diff
@@ -1,4 +1,5 @@
 import {
```

Comment:
One thing you'll have to consider is doing the same thing for our edit story page (once that's merged in). How flexible is this logic?

Reply:
Yeah for edit story we would do the same thing, I think the form is essentially the same it is just populated so we should be able to use the same logic to create the story object and pass that to the view story drawer